rules/windows/builtin/system/service_control_manager/win_system_hack_smbexec.yml

```
@@ -22,7 +22,7 @@ detection:
         Provider_Name: 'Service Control Manager'
         EventID: 7045
     selection_service_name:
-        ServiceName: 'BTOBTO'
+        ServiceName|re: '^[A-Za-z]{8}$'
```

```suggestion
        - ServiceName: 'BTOBTO'
        - ServiceName|re: '^[A-Za-z]{8}$'
```
Maybe we should keep both, so that it matches either the old fixed value and the new randomized pattern.

---

The regex in itself does not associate with smbexec alone as saying any random 8 char service name can lead to FP